### DISCUSSION TITLE
[Question] Compile Flet project

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#2027] [Q&A]
### Question

Good afternoon. I made a flet app, and I made many folders with files and elements that I use inside the project. I want to know how can I compile all of this in an EXE. Is better use pyinstaller or use Nuitka? and what considerations should I have so that all the folders and files remain in the exe and can run without problem? Thanks

![image](https://github.com/flet-dev/flet/assets/146734555/e6fbc455-9327-49ab-bddd-db2954199693)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
[`flet pack`](https://flet.dev/docs/cli/pack) (which uses pyinstaller for packaging), has a command named `--add-data` which you can use to pass files and folders which should be included in the compiled/package executable. 
More on that here:  https://www.pyinstaller.org/en/stable/spec-files.html#adding-data-files
